skyrl-tx/tx/tinker/config.py

```diff
@@ -22,9 +22,13 @@ class EngineConfig(BaseModel):
     max_lora_adapters: int = Field(default=32, description="Maximum number of LoRA adapters")
     max_lora_rank: int = Field(default=32, description="Maximum LoRA rank")
     tensor_parallel_size: int = Field(default=1, description="Tensor parallelism degree to use for the model")
-    micro_batch_size: int = Field(
+    train_micro_batch_size: int = Field(
         default=0,
-        description="Micro-batch size for gradient accumulation; 0 means disabled (use full batch)",
+        description="Micro-batch size in sequences for gradient accumulation; 0 means disabled (use full batch)",
```

Comment:
to make it clear, maybe `(measured in number of sequences)` instead of `in sequences`? Same comment for the other doc string below :)